### ISSUE TITLE
Concentric: transition properties separated

### ISSUE THREAD

**Issue by royaldark** [sasstools/sass-lint#964]
What version of Sass Lint are you using?
1.10.2

Please include any relevant parts of your configuration
```coffee
'property-sort-order': [
  1
  order: 'concentric'
]
```

With code like the following...
```scss
.some-class {
  transition: right 0.4s;
  transition-timing-function: ease-in-out;
  min-height: 500px;
}
```

...sass-lint complains that `min-height` and `transition-timing-function` should be swapped. This seems bizarre and counterintuitive.

In https://github.com/sasstools/sass-lint/blob/develop/lib/config/property-sort-orders/concentric.yml, many related properties are explicitly grouped (`flex` with `flex-*`, `animation` with `animation-*`, etc.), but `transition` properties are not.

It seems all possible `transition` properties should be grouped together for consistency.

Since the concentric property order seems to derive from https://github.com/brigade/scss-lint/blob/master/data/property-sort-orders/concentric.txt, I wasn't sure whether to post the issue here or with `scss-lint`.

**Comment by DanPurdy** [comment]
No, you were right to post it here. We just copied it from scss-lint in the first place as we wanted to closely match most of their rules / enforcements to make it easier all round for anyone to move between the projects.

The reason you aren't finding it in the order is because transition-timing-function is technically not implemented across all browsers yet
https://developer.mozilla.org/en/docs/Web/CSS/transition-timing-function

I'd be happy to open discussion about what properties should be added though and whether this is on such group that needs to be fleshed out.

**Comment by royaldark** [comment]
Ah, interesting. Well I notice all transition-related properties, including `transition` itself, are marked as experimental on MDN. And since `transition` is just short-hand for the other 4 rules, I'd argue the individual rules should be included as well.
